examples/erc721/src/lib.rs

```diff
@@ -151,4 +151,12 @@ impl Erc721Example {
 
         Ok(())
     }
+
+    pub fn pause(&mut self) -> Result<(), Error> {
```

Comment:
`Error` is inconsistent with other functions.

Reply:
I think it is fine. One less conversion to `Vec<u8>` in code.
cc @ggonzalez94

Reply:
I don't like `Vec<u8>`, but for consistency inside this example I would stick with it. It looks rare and might be confusing if inside the same file there are some functions that return `Result<(), Error> ` and others that return `Result<(), Vec<u8>>`even if they are equivalent.
We can do a refactor of all return types later as part of this [issue](https://github.com/OpenZeppelin/rust-contracts-stylus/issues/282). WDYT?

Reply:
I agree on refactoring later.